src/transport/mod.rs

```diff
@@ -27,25 +27,23 @@ use vm_memory::{ByteValued, VolatileSlice};
 
 use crate::BitmapSlice;
 
-mod file_traits;
-mod file_volatile_slice;
 mod fs_cache_req_handler;
 #[cfg(feature = "fusedev")]
 mod fusedev;
 #[cfg(feature = "virtiofs")]
 mod virtiofs;
 
-#[cfg(feature = "async-io")]
-pub use self::file_traits::AsyncFileReadWriteVolatile;
-pub use self::file_traits::{FileReadWriteVolatile, FileSetLen};
-#[cfg(feature = "async-io")]
-pub use self::file_volatile_slice::FileVolatileBuf;
-pub use self::file_volatile_slice::FileVolatileSlice;
 pub use self::fs_cache_req_handler::FsCacheReqHandler;
 #[cfg(feature = "fusedev")]
 pub use self::fusedev::{FuseBuf, FuseChannel, FuseDevWriter, FuseSession};
 #[cfg(feature = "virtiofs")]
 pub use self::virtiofs::VirtioFsWriter;
+#[cfg(feature = "async-io")]
+pub use dbs_fuse::buf::FileVolatileBuf;
```

Comment:
How about disabling conditional compilation for FileVolatileBuf?

Reply:
removed